src/components/UseCaseItem.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { Link, withRouter } from 'react-router'
+import { translate } from 'cozy-ui/react/I18n'
+
+// Fallback to get the item background image and avoid error if not found
+const getItemBackground = (figure, context) => {
+  let background = 'rgb(0, 130, 230)'
+  if (figure && context) {
+    try {
+      let img = require(`../contexts/${context}/assets/img/${figure}`)
+      background = `center/100% url(${img})`
+    } catch (e) {
+      background = 'rgb(0, 130, 230)'
```

Comment:
It is not *my* code as it is just a cleaner version of `KonnectorItem` but you're right, let's correct it now.